functions/src/services/GoogleSheet.ts

```diff
@@ -0,0 +1,358 @@
+const gapi = require('googleapis').google;
+
+export enum GoogleScopes {
+  SpreadSheets = 'https://www.googleapis.com/auth/spreadsheets',
+}
+
+const letters: string[] = [
+  'A',
+  'B',
+  'C',
+  'D',
+  'E',
+  'F',
+  'G',
+  'H',
+  'I',
+  'J',
+  'K',
+  'L',
+  'M',
+  'N',
+  'O',
+  'P',
+  'Q',
+  'R',
+  'S',
+  'T',
+  'U',
+  'V',
+  'X',
+  'Y',
+  'Z',
+];
+
+type ISheetId = string;
+
+export enum ISoundQualityReportSheet {
```

Comment:
I believe that any specific interfaces/enums etc related to some particular spreadsheet should be moved from this service into specific module. In this case into `SQR` module. This service should be spreadsheet-agnostic, as the `google-spreadsheet` package is.

Reply:
👍 Alright.

Reply:
If I understand correctly, moving the types to a separate file is a best practice. At least we follow this approach in frontend.